account_multicurrency_revaluation/wizard/wizard_currency_revaluation.py

```diff
@@ -72,7 +72,7 @@ def _get_default_revaluation_account_ids(self):
     revaluation_account_ids = fields.Many2many(
         comodel_name="account.account",
         string="Revaluation Accounts",
-        help="Accounts that will be revaluated",
+        help="Accounts that will be revaluated. ",
```

Comment:
This one only has one line so no need space.
```suggestion
        help="Accounts that will be revaluated.",
```